ansible/roles/test/tasks/link_flap_fib/link_flap_helper.yml

```diff
@@ -0,0 +1,57 @@
+- block:
+    - set_fact:
+        interface: "{{item}}"
+    
+    - debug: msg={{interface}}
+    
+    - set_fact:
+        peer_device: "{{neighbors[interface]['peerdevice']}}"
+        neighbor_interface: "{{neighbors[interface]['peerport']}}"
+    
+    - conn_graph_facts: host={{ peer_device }}
+      connection: local
+    
+    - set_fact:
+        peer_host: "{{device_info['mgmtip']}}"
+        peer_hwsku: "{{device_info['HwSku']}}"
+    
+    - set_fact:
+        down_port: "{{minigraph_port_indices[interface]}}"
+    
+    - name: Shut down neighbor interface {{neighbor_interface}} on {{peer_host}}
+      action: apswitch template=neighbor_interface_shut_single.j2
+      args:
+        host: "{{peer_host}}"
+        login: "{{switch_login[hwsku_map[peer_hwsku]]}}"
+      connection: switch
+    
+    - pause:
+        seconds: 20
+    
```

Comment:
maybe it is better to verify the port is down on the dut before you start the fib test.